### DISCUSSION TITLE
Switch yaml for json file

### DISCUSSION THREAD

**Discussion by AnaSofia-1294** [PaloAltoNetworks/docusaurus-openapi-docs#675] [General]
So I was generating my api documentation from yaml file, but now I want to change it into a json file. However this is giving me the following error:
[ERROR] TypeError: Cannot read properties of undefined (reading 'map')
Can someone help me please?

**Comment by htessaro** [comment]
Hi @AnaSofia-1294 , were you able to solve the problem?

**Reply by AnaSofia-1294** [reply]
I generated the json again and I could solve the problem... It was a wrong json formed that I was using hence this problem.
